### ISSUE TITLE
Having trouble getting dunit to work

### ISSUE THREAD

**Issue by therealbluepandabear** [intellij-dlanguage/intellij-dlanguage#824]
I added the dunit dependency, created some dummy tests, and then clicked on 'Process D Libraries', and I didn't see a run configuration for the tests that I wanted. 

I am unsure how I would run dunit tests using this plugin, could you add a tutorial in the `README.md` file... or possibly this is a bug, as it said in the `README.md` file: '...then run Process D Libraries you will be able to run d-unit tests' but this doesn't seem to be the case, I think the only way is to manually configure it to run the tests? :/

**Comment by SingingBush** [comment]
Are you using d-unit or dunit? See #431 as we don't support both. You should have a test run configuration available if the correct dependency is detected. It's worked for me in the past but I've not used it recently.

**Comment by therealbluepandabear** [comment]
Im using  d-unit, the one compatible with the plugin, no run configuration available

**Comment by therealbluepandabear** [comment]
Yeah after some time I could get it to detect the configuration but I got the same error as you did upon attempting to run the program.
